src/Control/Carrier/Parse/Measured.hs

```diff
@@ -1,17 +1,17 @@
 {-# LANGUAGE FlexibleContexts, FlexibleInstances, GADTs, GeneralizedNewtypeDeriving, MultiParamTypeClasses, RecordWildCards, TypeOperators, UndecidableInstances #-}
 -- | A carrier for 'Parse' effects suitable for use in production.
 module Control.Carrier.Parse.Measured
-( -- * Parse effect
-  module Control.Effect.Parse
-  -- * Parse carrier
-, ParseC(..)
+( -- * Parse carrier
+  ParseC(..)
   -- * Exceptions
 , ParserCancelled(..)
+  -- * Parse effect
+, module Control.Effect.Parse
```

Comment:
The standard idiom for carrier modules’ exports is now to export the effect at the end because that way the haddocks are a little better organized.